evalml/pipelines/components/transformers/imputers/time_series_imputer.py

```diff
@@ -165,6 +165,11 @@ def transform(self, X, y=None):
 
         if self._interpolate_cols is not None:
             X_interpolate = X.ww[self._interpolate_cols]
+            # Pandas' interpolate function doesn't work with IntegerNullable types at this time
```

Comment:
Let's put the link of the pandas issue in the comment
```suggestion
            # Pandas' interpolate function doesn't work with IntegerNullable types at this time
            # https://github.com/pandas-dev/pandas/issues/40252
```